code/formatters.py

```diff
@@ -228,15 +244,41 @@ def formatters_reformat_last(formatters: str) -> str:
         global last_phrase
         return format_phrase(last_phrase, formatters)
 
+    def insert_many(strings: List[str]) -> None:
+        """Insert a list of strings, sequentially."""
+        for string in strings:
+            actions.insert(string)
+
 
 @ctx.capture(rule="{self.formatters}+")
 def formatters(m):
     return ",".join(m.formatters_list)
 
 
-@ctx.capture(rule="<self.formatters> <user.text>")
+@ctx.capture(
+    # Add anything else into this that you want to be able to speak during a
+    # formatter.
+    rule="(<user.symbol> | <user.letter> | <user.number>)"
```

Comment:
This needs to be changed to use <user.number_key> and <user.symbol_key> now that #240 has been merged.

Reply:
Ok - I'll gate "number" with a prefix too, that seems to be the consensus. I'll leave letters in for now, you guys experiment with it and gate that too if it becomes a problem.